<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionDlg</class>
 <widget class="QDialog" name="CollectionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#CollectionDlg {
background-color:rgb(233, 195, 120);
border-image: url(:/pictures/lottery_zoom_frame.png);
};
font: 20pt &quot;方正手迹-旅行青蛙体 简&quot;;</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#title{
font: 18pt &quot;华文彩云&quot;;}</string>
   </property>
   <property name="text">
    <string>奖品图鉴</string>
   </property>
  </widget>
  <widget class="QWidget" name="Yellow" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/lottery_ball_01.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="Red" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/lottery_ball_02.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="Blue" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/lottery_ball_03.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="Green" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/lottery_ball_04.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="White" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/lottery_ball_05.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="YellowLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RedLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BlueLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="GreenLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WhiteLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="ok_btn" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/icon/ok.png);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
